lib/cli/src/generators/configure.ts

```diff
@@ -0,0 +1,24 @@
+import fse from 'fs-extra';
+
+function mainConfigurationGenerator(addons?: string[], custom?: any) {
+  const hasSrc = fse.existsSync('./src');
+
+  const config = {
+    stories: [!hasSrc && '../stories/**/*.stories.*', hasSrc && '../src/**/*.stories.*'].filter(
+      Boolean
+    ),
+    addons: [
+      '@storybook/addon-actions',
+      '@storybook/addon-links',
+      '@storybook/addon-docs',
```

Comment:
Shouldn't it be `@storybook/addon-essentials` instead?

Reply:
See https://github.com/storybookjs/storybook/pull/10913/commits/2570a66ae08d16e6110823627878258740785a96